applications/openshift/accounts/accounts_no_clusterrolebindings_default_service_account/tests/clusterrolebindings_other_serviceaccounts.pass.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+# remediation = none
+
+yum install -y jq
```

Comment:
try to use instead:
`# packages = jq`

that makes it agnostic to the package manager.

Reply:
Thanks for the suggestion. I changed it on all occurences.